crates/graphql/src/schema/objects/marketplace.rs

```diff
@@ -85,7 +85,21 @@ impl Marketplace {
     pub async fn auction_house(&self, context: &AppContext) -> FieldResult<Option<AuctionHouse>> {
         context
             .auction_house_loader
-            .load(self.auction_house_address.clone().into())
+            .load(
+                self.auction_house_address
+                    .as_deref()
+                    .unwrap_or_default()
+                    .to_owned()
+                    .into(),
```

Comment:
`*self.auction_house_address` should be type `PublicKey<AuctionHouse>` That should be the input for this loader.